bindings/java/mac_run.me

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+cd `dirname $0`
+
+#unset _JAVA_OPTIONS
+if [ "x$JAVA_HOME" = "x" ]; then
+    JAVA_HOME=`/usr/libexec/java_home`
+fi
+
+if [ ! -f ../libblst.a -o ../blst.h -nt ../libblst.a ]; then
+    (cd ..; ../build.sh)
+fi
+
+if [ ! -f blst_wrap.cpp -o ../blst.swg -nt blst_wrap.cpp \
+                        -o ../blst.hpp -nt blst_wrap.cpp \
+                        -o ../blst.h   -nt blst_wrap.cpp ]; then
+    (set -x; swig -c++ -java -outdir . -o blst_wrap.cpp ../blst.swg)
+fi
+
+if [ ! -f libblst.dylib -o blst_wrap.cpp -nt libblst.dylib \
+                     -o ../libblst.a  -nt libblst.dylib ]; then
+    (set -x; c++ -march=native -shared -o libblst.dylib -fPIC \
```

Comment:
`-march=native` makes no sense here, a least it won't buy you anything. Or is there some rationale other than performance behind this?

Side note about .dylib extension. There are suggestions that it should be .jnilib. It's not impossible that both are recognized. This however asks for question if there is possibility that one won't be recognized in the future.